public/config.json

```diff
@@ -1,3 +1,21 @@
 {
-    "COMMON_PATH": ""
+    "DARK_MODE": "auto",
+    "SWITCH_COLORS": false,
+    "NOTATION_24H": true,
+    "WRAP_HEADER": false,
+    "COMMON_PATH": [],
+    "COLUMNS": [
+        "Name",
+        "Progress",
+        "ETA",
+        "Download Speed",
+        "Upload Speed",
+        "File Size",
+        "Downloaded",
+        "Uploaded",
+        "Downloading from",
+        "Seeding to"
+    ],
```

Comment:
Hmm... Columns have ID's. Ideally we should use ID's here, but it would make configuration a bit more complicated because the user would have to map the ID's to the actual names.

I implemented ID's because that would would allow me to change the naming of the column without breaking any kind of browser configuration, but this will break that pattern again.

Reply:
Currently the best i can think off (until you want to localize the interface) and even then its probably best to stick to english as the configuration "language" instead of integer.

Reply:
I had an idea which I documented here: https://github.com/johman10/flood-for-transmission/pull/514#discussion_r1170326281. This is not just about localization, but also about renaming of columns in case it turns out a column name is unclear. This has happened before with:
https://github.com/johman10/flood-for-transmission/issues/336 and made me switch to using ID's as a part of the solution in https://github.com/johman10/flood-for-transmission/pull/342